tests_integration/__tests__/loglevel.js

```diff
@@ -1,21 +1,22 @@
 "use strict";
 
 const runPrettier = require("../runPrettier");
+const stripAnsi = require("strip-ansi");
 
 test("do not show logs with --loglevel silent", () => {
   runPrettierWithLogLevel("silent", null);
 });
 
 test("do not show warnings with --loglevel error", () => {
-  runPrettierWithLogLevel("error", ["[error]"]);
+  runPrettierWithLogLevel("error", [/\[error]/]);
```

Comment:
@thorn0 do you think I should restore this to string?

Reply:
I have no idea why those tests failed. It's really difficult to read them because their output is cluttered with uninterpreted color codes. But if you don't mind, let's try to restore it and see what happens.

Reply:
The failure reason already found, on macOS `[error]` is colored, so I use stripAnsi remove color first.

this change doesn't mater anymore, `string` also passing

Reply:
Great. Let's revert this change then.

Reply:
sure